pages/home.page.ts

```diff
@@ -3,9 +3,19 @@ import { Locator, Page } from "@playwright/test"
 export class HomePage {
     page: Page;
     combinationPliersProduct: Locator;
+    slipJointPliersProduct: Locator;
+    sortingButton: Locator;
+    productName: Locator;
+    productPrice: Locator;
+    sanderLabel: Locator;
 
     constructor(page: Page) {
         this.page = page;
-        this.combinationPliersProduct = this.page.getByText("Combination Pliers")
+        this.combinationPliersProduct = this.page.locator('[data-test="product-name"]:has-text("Combination Pliers")');
```

Comment:
краще створити метод:
```suggestion
        getProductByText(text: string) {
                return this.page.locator(`[data-test="product-name"]:has-text("${text}")`);
        }
```